patrickfuchs/buildH
# [WIP] Create Python package

Here is an attempt to create a Python package for Pypi.

To test this PR, you can:

1. Clone this fork:
```
$ https://github.com/pierrepo/buildH.git buildh-tmp
$ cd buildh-tmp
$ git checkout create-package
```

2. Build and load conda environment:
```
$ conda env create -f binder/environment.yml
$ conda activate buildh
```

3. Install buildH in the conda env:
```
$ pip install -e .
```


4. Run buildH
```
$ buildH
```

@HubLot could you check relative imports (`from . import toto`) are correct?

Tests are expected to fail.
